modules/client/src/node.ts

```diff
@@ -279,10 +279,13 @@ export class NodeApiClient implements INodeApiClient {
         `Must have instantiated a channel provider (ie a signing thing) before setting auth token`,
       );
     }
+    // TODO: merge:
+    // https://github.com/WalletConnect/walletconnect-monorepo/pull/210
+    // and publish / update package
     const nonce = await this.send("auth.getNonce", {
       address: this.channelProvider.signerAddress,
     });
```

Comment:
How is this TODO comment related to `auth.getNonce`?

Reply:
see above:

> The channel provider (when injected) would not allow access to this.channelProvider.signerAddress until the latest wc is published